search-helpers/bbl.js

```diff
@@ -20,13 +20,17 @@ const bbl = (string) => {
      LIMIT 5
   `;
 
-  return carto.SQL(tenDigits ? SQL : 'SELECT 1 LIMIT 0').then(rows =>
-    rows.map((row) => {
-      row.label = toTitleCase(row.address);
-      row.type = 'lot';
-      delete row.address;
-      return row;
-    }));
+  if (tenDigits) {
+    return carto.SQL(SQL).then(rows =>
```

Comment:
carto.SQL does not necessarily return something (see: https://github.com/NYCPlanning/labs-search-api/blob/32-invalid-requests-to-carto/utils/carto.js#L31 -- in the catch block it just logs, with no return)

either make the carto.SQL function always return something  OR check that rows exists/is iterable before calling `rows.map` 

might as well make this fix too while we're in here!

Reply:
will do - thanks!